scene/main/node.cpp

```diff
@@ -4155,6 +4155,11 @@ int Node::get_persistent_signal_connection_count() const {
 	return Object::get_persistent_signal_connection_count();
 }
 
+uint32_t Node::get_signal_connection_flags(StringName const &p_signal, Callable const &p_callable) const {
```

Comment:
```suggestion
uint32_t Node::get_signal_connection_flags(const StringName &p_signal, const Callable &p_callable) const {
```
This should have been caught by the static checks, unsure why it wasn't